### ISSUE TITLE
[Orpheus] Returning InternalServerError

### ISSUE THREAD

**Issue by garfield69** [Jackett/Jackett#13711]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Using HTTP Client: HttpWebClient2
Using FlareSolverr: http://127.0.0.1:8191
Using proxy: Disabled
App config/log directory: C:\ProgramData\Jackett
ThreadPool MaxThreads: 1023 workerThreads, 1000 completionPortThreads
Running in Docker: No
Jackett variant: CoreWindows
OS version: Microsoft Windows NT 10.0.19043.0 (64bit OS)
Environment version: 6.0.11 (C:\ProgramData\Jackett\)
Starting Jackett v0.20.2236
```

### Description

TEST fails with error `Request to Orpheus failed with status InternalServerError.`
see log for details

### Logged Error Messages

```text
2022-11-12 16:28:39	Warn	
Request to Orpheus failed with status InternalServerError. Retrying in 1s... (Attempt 2 of 2).
2022-11-12 16:28:40	Debug	
WebClient(HttpWebClient2).GetResultAsync(Method: GET Url: https://orpheus.network/ajax.php?action=browse&order_by=time&order_way=desc)
2022-11-12 16:28:41	Debug	
WebClient(HttpWebClient2): Returning InternalServerError => 0 bytes
2022-11-12 16:28:41	Error	
Error on 20221112162841 for Orpheus.txtNewtonsoft.Json.JsonReaderException: Error reading JObject from JsonReader. Path '', line 0, position 0.
   at Newtonsoft.Json.Linq.JObject.Load(JsonReader reader, JsonLoadSettings settings)
   at Newtonsoft.Json.Linq.JObject.Parse(String json, JsonLoadSettings settings)
   at Jackett.Common.Indexers.Abstract.GazelleTracker.PerformQuery(TorznabQuery query) in D:\a\1\s\src\Jackett.Common\Indexers\Abstract\GazelleTracker.cs:line 221
2022-11-12 16:28:41	Info	
Executed action Jackett.Server.Controllers.IndexerApiController.Test (JackettConsole) in 4182.858ms
2022-11-12 16:28:41	Info	
Executed endpoint 'Jackett.Server.Controllers.IndexerApiController.Test (JackettConsole)'
2022-11-12 16:28:41	Error	
Jackett.Common.IndexerException: Exception (orpheus): Error reading JObject from JsonReader. Path '', line 0, position 0.
 ---> System.Exception: Parse error
 ---> Newtonsoft.Json.JsonReaderException: Error reading JObject from JsonReader. Path '', line 0, position 0.
   at Newtonsoft.Json.Linq.JObject.Load(JsonReader reader, JsonLoadSettings settings)
   at Newtonsoft.Json.Linq.JObject.Parse(String json, JsonLoadSettings settings)
   at Jackett.Common.Indexers.Abstract.GazelleTracker.PerformQuery(TorznabQuery query) in D:\a\1\s\src\Jackett.Common\Indexers\Abstract\GazelleTracker.cs:line 221
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseWebIndexer.OnParseError(String results, Exception ex) in D:\a\1\s\src\Jackett.Common\Indexers\BaseIndexer.cs:line 718
   at Jackett.Common.Indexers.Abstract.GazelleTracker.PerformQuery(TorznabQuery query) in D:\a\1\s\src\Jackett.Common\Indexers\Abstract\GazelleTracker.cs:line 298
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query, Boolean isMetaIndexer) in D:\a\1\s\src\Jackett.Common\Indexers\BaseIndexer.cs:line 324
   --- End of inner exception stack trace ---
   at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query, Boolean isMetaIndexer) in D:\a\1\s\src\Jackett.Common\Indexers\BaseIndexer.cs:line 324
   at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query, Boolean isMetaIndexer) in D:\a\1\s\src\Jackett.Common\Indexers\BaseIndexer.cs:line 695
   at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in D:\a\1\s\src\Jackett.Common\Services\IndexerManagerService.cs:line 316
   at Jackett.Server.Controllers.IndexerApiController.Test() in D:\a\1\s\src\Jackett.Server\Controllers\IndexerApiController.cs:line 147
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in D:\a\1\s\src\Jackett.Server\Middleware\CustomExceptionHandler.cs:line 61
```

**Comment by petermuffad** [comment]
Can confirm, Orpheus not working for me too!

```
An error occurred while testing this indexer
Exception (orpheus): Error reading JObject from JsonReader. Path '', line 0, position 0.: Parse error
```

**Comment by SawkeeReemo** [comment]
Why is this marked as "Down / Waiting for removal"? What does that mean? Because Orpheus is not down. Curious...

**Comment by ilike2burnthing** [comment]
Their API is currently non-functional, so as it pertains to Jackett the tracker is 'down'.

The label is used so we can monitor trackers with issues. Typically, an issue lasting more than 30 days (without sign of some resolution in the works) results in the indexer being removed.

I very much doubt Orpheus will be removed.

**Comment by SawkeeReemo** [comment]
Ah, good to know. Thank you for the clarification.

**Comment by Mannekino** [comment]
Any idea when this will work again?

**Comment by garfield69** [comment]
the web site is down for maintenance so perhaps it is being worked on right now.

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://orpheus.network/ajax.php?action=browse&order_by=time&order_way=desc Returning InternalServerError => 0 bytes

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/13711#issuecomment-1575107565

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/13711#issuecomment-1575107565

**Comment by garfield69** [comment]
online
